rpm/sailjail.spec

```diff
@@ -81,3 +120,13 @@ make HAVE_FIREJAIL=%{jailfish} -C unit test
 %files tools
 %defattr(-,root,root,-)
 %{_bindir}/measure_launch_time
+
+%files daemon
+%defattr(-,root,root,-)
+%license COPYING
+%{_bindir}/sailjaild
+%{_unitdir}/*.service
+%{_unitdir}/multi-user.target.wants/*.service
+%config %{_sysconfdir}/dbus-1/system.d/sailjaild.conf
+%attr(0750,root,root) %dir %{_sharedstatedir}/sailjail/settings
```

Comment:
This is a little problematic from encryption point of view: Enabling user data encryption, which is what happens on first boot, will destroy all the data in /home including anything that package management may have created.

Elsewhere (connman, nemo-qml-plugin-settings) these directories are created on demand. This should probably do that instead.

Reply:
Create dirs on sailjaild startup and change dirs in spec to be ghosts.